docs/admin/accessing-the-api.md

```diff
@@ -57,43 +57,39 @@ users in its object store.
 
 ## Authorization
```

Comment:
No.
I moved the detailed information about authorization to this page: 
    - docs/admin/authorization/index.md
However, the topic needs to remain here because this is part of the flow discussed on this page. I included high-level information for authorization and a link to the page that includes details.